src/Makefile.am

```diff
@@ -455,6 +455,7 @@ libbitcoin_consensus_a_SOURCES = \
   arith_uint256.h \
   bobtail/bobtail.cpp \
   bobtail/bobtail.h \
+  bobtail/bobtailblock.h \
```

Comment:
Can we just call this bobtail/block.h without any worry of ambiguity? I did the same for bobtail/validation.cpp

Reply:
sure